packages/contractkit/src/address-registry.ts

```diff
@@ -22,8 +21,8 @@ export class AddressRegistry {
   async addressFor(contract: CeloContract): Promise<Address> {
     if (!this.cache.has(contract)) {
       debug('Fetching address from Registry for %s', contract)
-      const hash = Web3.utils.soliditySha3({ type: 'string', value: contract })
-      const address = await this.registry.methods.getAddressFor(hash).call()
+      // const hash = Web3.utils.soliditySha3({ type: 'string', value: contract })
```

Comment:
why comment this out?

Reply:
Thanks for catching this.. my commit included some other work that should not be here.

Now that's fixed